web/src/user_profile.js

```diff
@@ -798,6 +798,16 @@ export function show_edit_user_info_modal(user_id, $container) {
         user_deactivation_ui.confirm_reactivation(user_id, handle_confirm, true);
     });
 
+    $("#edit-user-form #edit_user_full_name").on("input", () => {
+        const $full_name = $("input[name='full_name']");
+        const $submit_btn = $("#user-profile-modal .dialog_submit_button");
+
+        const full_name_empty = $full_name.val().trim() === "";
+
+        $full_name.toggleClass("invalid", full_name_empty);
+        $submit_btn.prop("disabled", full_name_empty);
```

Comment:
Is there other logic that controls whether the submit button is disabled related to other inputs? We need to be careful if so.

Reply:
We currently disable the button after clicking on it to save the changes. So, would need to handle the case if the button is disabled with spinner shown on it after clicking on the button and then user changes the name. 

I guess there is another PR which disables the button if no field is changed, so we should make sure to rebase the other PR once any of these is merged.

Reply:
I can also check if the button is disabled or disabled with spinner before disabling/enabling it? Would that work?

Reply:
I guess just having a check before enabling the button would be enough.